modules/nw-sriov-about-all-multi-cast-mode.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-interface-sysctl-sriov-device.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-about-all-multi-cast-mode_{context}"]
+= About all-multicast mode
+
+Enabling all-multicast mode, particularly in the context of rootless applications, is important as without it, you would be required to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC).
```

Comment:
Why is it preferred to enable all-multicast mode over granting `NET_ADMIN` capability to the pod's SSecurity Context Constraints?

Reply:
This is primarily associated with security concerns enabling NET_ADMIN capability provides extensive control over networking, including multi-cast, which might not be necessary for the pod's intended functionality. Allowing NET_ADMIN capability grants the pod privileges to make changes that extend beyond its specific requirements, potentially exposing security vulnerabilities.

Reply:
I did add a clarification